### ISSUE TITLE
.shebang.bak file blocking extension activation

### ISSUE THREAD

**Issue by lee218llnl** [LLNL/spack#1043]
$ spack activate py-basemap 

...
==> Error: Extension blocked by file: /collab/usr/gapps/python/build/spack-toss3/opt/spack/toss_3_x86_64_ib/gcc-4.8.5/python-2.7.11-yuzawyngcavhb6ff5rumbc7qmz43cyzw/bin/f2py.shebang.bak

Python's package.py has:
```

        if ext_pkg.name != 'py-numpy':
            patterns.append(r'bin/f2py$')
```

but this doesn't account for https://github.com/LLNL/spack/blob/develop/lib/spack/spack/hooks/sbang.py#L61

**Comment by tgamblin** [comment]
Hm.  We could either just not generate the backup (sbang seems to have been pretty stable) or we could add it to the things ignored by default.  Thoughts?

**Comment by lee218llnl** [comment]
I wouldn't mind if we did not generate the backup. One could easily just delete the first line of the output file if they want the original.

**Comment by tgamblin** [comment]
Agree.  The backup was just there out of an overabundance of caution.

**Comment by adamjstewart** [comment]
I believe this issue can be closed now.

**Comment by tgamblin** [comment]
Fixed by #1064
